doc/use_cli_in_airgapped_clouds.md

```diff
@@ -0,0 +1,36 @@
+# Use Azure CLI in Airgapped Clouds
+
+## Introduction
+While Azure CLI is not fully supported in an airgapped cloud which has no public internet connectivity, most commands could run without issues if the underlying service is supported in that cloud.
```

Comment:
air-gapped or air gapped

Reply:
"air-gapped". "air gapped" is not grammatically correct.

Reply:
In some internal wiki, it's called `airgapped clouds`, so I followed the spelling.